i3lock.c

```diff
@@ -2195,9 +2206,10 @@ int main(int argc, char *argv[]) {
                 cairo_rectangle(ctx, 0, 0, last_resolution[0], last_resolution[1]);
                 cairo_fill(ctx);
                 cairo_pattern_destroy(pattern);
+            } else {
+                cairo_set_source_surface(ctx, img, 0, 0);
+                cairo_paint(ctx);
             }
-            cairo_set_source_surface(ctx, img, 0, 0);
-            cairo_paint(ctx);
```

Comment:
Originally if tile was set then `cairo_set_source_surface(ctx, img, 0, 0);` and `cairo_paint(ctx);` would occur, but now it wouldn't? Should we duplicate those two lines inside `else if (tile)` as well?

Reply:
Nevermind I see this was mitigated in a later commit